src/styles/blocks/card.scss

```diff
@@ -0,0 +1,76 @@
+.card {
+  width: 200px;
+  border: 1px solid $color-gray;
+  padding: 32px 16px 16px;
+
+  &__description {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 4px;
+    color: $main-color;
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondary-color;
```

Comment:
Ensure that the variable `$secondary-color` is defined in your SCSS files. If it's not defined, this will cause a compilation error.